src/scripts/browserDeviceProfile.js

```diff
@@ -176,6 +176,21 @@ define(['browser'], function (browser) {
         return false;
     }
 
+    //temporary, until canPlayType(av1) is fixed upstream for chrome
+    function testCanPlayAv1() {
+        if (browser.orsay) {
+            return false;
+        } else if (browser.tizenVersion >= 5.5) {
+            return true;
+        } else if (browser.web0sVersion >= 5 && window.outerHeight >= 2160) {
+            return true;
+        } else if (browser.chrome && browser.versionMajor >= 69) {
+            return true;
+        }
+
+        return false;
+    }
```

Comment:
```suggestion
    function testCanPlayAv1() {
        if (browser.tizenVersion >= 5.5) {
            return true;
        } else if (browser.web0sVersion >= 5 && window.outerHeight >= 2160) {
            return true;
        }
        
        return videoTestElement.canPlayType('video/webm; codecs="av01.0.15M.10"').replace(/no/, '');
    }
```

Reply:
Also please remove the line with `var canPlayAv1 = videoTestElement.canPlayType('video/webm; codecs="av1"').replace(/no/, '');` since it's now unused.